src/tetris/profiler/profile.ts

```diff
@@ -35,6 +65,7 @@ export default class Profile {
 		this._data = new Map<string, BaseProfileData>();
 		this._newProfileData<GeoLocation>(LOCATION_KEY);
 		this._newProfileData<number>(AGE_KEY).updateStrategy = new NumberDataUpdateStrategy();
+		this._newProfileData<string>(ETHNICITY_KEY);
```

Comment:
It seems like SkinHealth, Acne etc. are not in the profile, is that correct?

Reply:
That is correct. I fixed that within the latest commit.